packages/ui-components/src/components/search/search.tsx

```diff
@@ -38,6 +38,7 @@ export class KvSearch implements ISearchEvents, ITextFieldEvents {
 
 	private onTextChange = (event: CustomEvent<string>) => {
 		this._value = event.detail;
+		this.textChange.emit(this.value);
```

Comment:
For some reason the first time we type something, the emitted value is null 🤷‍♂️ 

Please investigate this issue.